### ISSUE TITLE
The installation guide for Pynodestatviz is out off date

### ISSUE THREAD

**Issue by blackhawk101x** [adjacentlink/emane-tutorial#18]
Step 3, build and install the pytnodestatviz, the make file does not exist. Since the readme in the pytnodestatviz points to this project for installation instructions, I opened the issue here. The steps appear to be:

```
# git clone https://github.com/adjacentlink/pynodestatviz
# cd pynodestatviz
# ./autogen.sh
# ./configure
# sudo yum install ./.rpmbuild/RPMS/noarch/*.rpm
```

**Comment by blackhawk101x** [comment]
This is directed at the Redhat/CentOS platforms

**Comment by sgalgano** [comment]
Thanks. I updated the build instructions accordingly.

**Comment by ajeyaajeya** [comment]
it did not work on Ubuntu 16.04
`ajeya@AJ-16046LTS:~/Downloads/pynodestatviz$ cd pynodestatviz/
ajeya@AJ-16046LTS:~/Downloads/pynodestatviz/pynodestatviz$ ls
__init__.py           nodepropertyframe.py  renderexception.py
nodedisplaycanvas.py  nodestatviz.py        schema
ajeya@AJ-16046LTS:~/Downloads/pynodestatviz/pynodestatviz$ cd ..
ajeya@AJ-16046LTS:~/Downloads/pynodestatviz$ ./autogen.sh 
./autogen.sh: line 76: aclocal: command not found
./autogen.sh: line 78: automake: command not found
./autogen.sh: line 80: autoreconf: command not found
ajeya@AJ-16046LTS:~/Downloads/pynodestatviz$ sudo apt-get install aclocal
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package aclocal
`

how to resolve this?

**Comment by sgalgano** [comment]
It looks like you are missing some packages on your system. Try adding these:

```
sudo apt-get install autoconf automake libtool 
```

**Comment by PVineeth** [comment]
> ubuntu:~/Documents/pynodestatviz/.debbuild$ sudo apt-get install ./*pynodestatviz*.deb
> 
> Some packages could not be installed. This may mean that you have
> requested an impossible situation or if you are using the unstable
> distribution that some required packages have not yet been created
> or been moved out of Incoming.
> The following information may help to resolve the situation:
> 
> The following packages have unmet dependencies:
>  python3-pynodestatviz : Depends: python3-pmw but it is not installable
> E: Unable to correct problems, you have held broken packages.
> 

python3-pmw package is not available on Ubuntu 18.04/20.04 LTS.

**Comment by ehengberg** [comment]
When I  try and build the rpm's I get this tar error:

make rpm
if test -f pynodestatviz.spec; then chmod u+w pynodestatviz.spec; fi
sed -e 's|@VERSION[@]|0.2.3|g' -e 's|@DEBIAN_VERSION[@]||g' -e 's|@DATE_RFC2822[@]|Mon, 04 Oct 2021 14:00:08 -0400|g' pynodestatviz.spec.in > pynodestatviz.spec
chmod g-w,u-w pynodestatviz.spec
if test -d "pynodestatviz-0.2.3"; then find "pynodestatviz-0.2.3" -type d ! -perm -200 -exec chmod u+w {} ';' && rm -rf "pynodestatviz-0.2.3" || { sleep 5 && rm -rf "pynodestatviz-0.2.3"; }; else :; fi
test -d "pynodestatviz-0.2.3" || mkdir "pynodestatviz-0.2.3"
test -n "" \
|| find "pynodestatviz-0.2.3" -type d ! -perm -755 \
	-exec chmod u+rwx,go+rx {} \; -o \
  ! -type d ! -perm -444 -links 1 -exec chmod a+r {} \; -o \
  ! -type d ! -perm -400 -exec chmod a+r {} \; -o \
  ! -type d ! -perm -444 -exec /bin/sh /local_disk/ER30730/pynodestatviz/install-sh -c -m a+r {} {} \; \
|| chmod -R a+r "pynodestatviz-0.2.3"
tardir=pynodestatviz-0.2.3 && ${TAR-tar} chof - "$tardir" | GZIP=--best gzip -c >pynodestatviz-0.2.3.tar.gz
tar: value 50020745 out of uid_t range 0..2097151
tar: Exiting with failure status due to previous errors
